### DISCUSSION TITLE
fan speed

### DISCUSSION THREAD

**Discussion by Justinnine** [mriscoc/Ender3V2S1#826] [Q&A]
In version Ender3s1-F1-UBL-MPC-20221222.bn is the fan speed supposed to increase by it's self when printing?  and should I keep lowering it to 100 percent?

**Comment by mriscoc** [comment]
There are two fans.

**Reply by Justinnine** [reply]
Yes, I know but the fan that cools the print keeps speeding up to 225 and I have to go into tuning mode to lower it back down to 100%.  Is it normal for it to speed up and should I just let it go or keep lowering it back down to the 100% value?

**Reply by dfinberg** [reply]
The speed is an 8 bit unsigned value, so goes from 0-255.  255 is the max, i.e. 100%.  You're reducing it down to less than half the normal speed.

If you did actually want to reduce the speed, you can set it in your slicer, but you probably don't want to do that.

**Comment by neshami** [comment]
255 is 100%

Sendt fra Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Justinnine** [comment]
So I should just let it set it's self to 255 then?

Justin
________________________________

**Reply by mriscoc** [reply]
![image](https://user-images.githubusercontent.com/2745567/228674946-efb98a63-60c2-45e4-b7ea-0e7acad003f4.png)
